pjsip-apps/src/pjsua/pjsua_app_config.c

```diff
@@ -148,7 +148,7 @@ static void usage(void)
     puts  ("  --quality=N         Specify media quality (0-10, default="
     				  xstr(PJSUA_DEFAULT_CODEC_QUALITY) ")");
     puts  ("  --ptime=MSEC        Override codec ptime to MSEC (default=specific)");
-    puts  ("  --no-vad            Disable VAD/silence detector (default=vad enabled)");
+    puts  ("  --no-vad            Disable VAD/silence detector (default=specific)");
```

Comment:
if there is room to put "codec specific", better put the complete description, IMO (also for --ptime above).

Reply:
Actually the VAD setting in PJSUA affects both codec & sound device (PJSUA will add/remove `PJMEDIA_AUD_DEV_CAP_VAD` flag). But as VAD in audio device seems to only applicable to non-PCM mode (e.g: Symbian APS direct), so more or less it is codec related too.